share/python/inputparser.py

```diff
@@ -666,8 +664,8 @@ def process_input(raw_input, print_level=1):
     temp = re.sub(comment, '#', temp)
 
     # Check the brackets and parentheses match up, as long as this is not a pickle input file
-    if not re.search(r'pickle_kw', temp):
-        check_parentheses_and_brackets(temp, 1)
+    #if not re.search(r'pickle_kw', temp):
+    #    check_parentheses_and_brackets(temp, 1)
```

Comment:
Ill change it back, but in the input I needed to do things like `str += """(start of paren""" which is not allowed with this.